app/src/main/java/jimpatrizi/com/netrtl/ExecuteButtonOnClickListener.java

```diff
@@ -41,12 +41,20 @@ public void onClick(View view) {
             //for all parameter values, get their respective daemoncallbackstrings and send thosse individually
             //to the server, server handles \n between each string
             for (Parameters p : Parameters.values()) {
+
+                // Volume is sent to the server when changed
+                if (p.equals(Parameters.VOLUME))
```

Comment:
Resolves the double receive bug after pressing EXECUTE

Reply:
ahhh it was vol causing that